app/src/test/java/com/example/mymovieinfo/MovieIntegrationTest.kt

```diff
@@ -8,7 +8,7 @@ import org.junit.Assert.*
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
-class ExampleUnitTest {
+class MovieIntegrationTest {
 
```

Comment:
Renamed 'ExampleUnitTest' to 'MovieIntegrationTest'. Currently, there isn't any unit testing going on and based on the example test you guys have, I assume integration tests will go here in the future, so I thought the name 'MovieIntegrationTest' fit better.

Reply:
Good idea... provided there are tests in here.